apps/docs/pages/guides/storage/image-transformations.mdx

```diff
@@ -59,6 +59,32 @@ supabase.storage.from('bucket').download('image.jpg', {
 })
 ```
 
+## Automatic Image Optimisation (WebP)
+
+When using the image transformation API we will automatically find the best format supported by the browser and return that to the client, without any code change.
+For instance, if you use Chrome when viewing a jpeg image and using transformation options, you'll see that the content-type returned is `webp`.
+
+In result, this will lower the bandwidth that you send to your users and your application will load much faster.
+
+<Admonition type="note">
+    We currently only support WebP. AVIF support will come in the near future
+</Admonition>
+
+**Disabling automatic optimisation:**
+
+In case you'd like to return the original format of the image and **opt-out** from the automatic image optimization detection, you can pass the `format=origin` parameter when requesting a transformed image,
+this is also supported in the SDK starting from v2.2.0
```

Comment:
```suggestion
this is also supported in the JavaScript SDK starting from v2.2.0
```

Reply:
Note to @inian to update the docs. cc @dshukertjr for updating the dart lib with this option